openedx/features/job_board/views.py

```diff
@@ -1,9 +1,21 @@
+from django.views.generic.list import ListView
 from edxmako.shortcuts import render_to_response
 
+from .models import Job
 
-def list_jobs(request):
 
-    return render_to_response('features/job_board/job_list.html', {})
+class JobsListView(ListView):
+
+    model = Job
+    context_object_name = 'jobs_list'
+    paginate_by = 10
+    template_name = 'features/job_board/job_list.html'
+    ordering = ['-created']
```

Comment:
How about using ordering = ['-created'] in model's Meta class? I think it is more relevant to model rather  than views. WDYT?

Reply:
As per my understanding, adding the `ordering` in model Meta class could force an ordering every time we fetch records from the database. This is not something we necessarily want each and every time. Putting such a restriction at the model level does not sound very good.

Reply:
Actually jobs are something we always need latest jobs despite the fact where we are fetching. By putting this in model will populate admin side according to ordering we are displaying for learner/user.. means latest.. It will help admins to see latest jobs on top. 

Just another thought, how about using modified instead of created? As of now it ll behave same but if we give option to edit jobs after mvp it will show modified jobs on top.

Reply:
Please have a look on it again

Reply:
The admin thing is not a current client requirement and I am not comfortable  making this call at this point with a lot of unknowns. Same goes for the `modified`. Let's discuss this again once the the future requirements are clear.